dbfit-java/core/src/main/java/dbfit/util/FitNesseTestHost.java

```diff
@@ -9,6 +10,7 @@
 
     static {
         TypeAdapter.registerParseDelegate(BigDecimal.class, BigDecimalParseDelegate.class);
+        TypeAdapter.registerParseDelegate(BigInteger.class, BigIntegerParseDelegate.class);
```

Comment:
The range of values of `java.math.BigInteger` I think is wider than `BIGINT UNSIGNED`.